ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-release-ocm-2.6.yaml

```diff
@@ -152,6 +160,15 @@ tests:
         SERVICE_BASE_REF=release-ocm-2.6
         NUM_MASTERS=1
     workflow: assisted-baremetal
+- as: e2e-ai-operator-ztp
```

Comment:
can you apply ``skip_if_only_changed/run_if_changed`` configurations?

Reply:
done for 2.6.
It's optional on master, so we don't need it